e2e/atlas/helper_test.go

```diff
@@ -240,6 +243,21 @@ func RandProjectNameWithPrefix(prefix string) (string, error) {
 	return prefixedName, nil
 }
 
+func MongoDBMajorVersion() (string, error) {
+	atlasClient := mongodbatlas.NewClient(nil)
+	atlasURL := os.Getenv("MCLI_OPS_MANAGER_URL")
+	if atlasURL != mcliOpsManagerURL {
```

Comment:
[nit] what if I actually want to run the tests against production or my local instance, the main point of checking the URL is parsable is to catch human error/typos, not to forbid the use of any other url that's not dev